config/config.go

```diff
@@ -217,7 +216,7 @@ type ScrapeConfig struct {
 	// Indicator whether the scraped metrics should remain unmodified.
 	HonorLabels bool `yaml:"honor_labels,omitempty"`
 	// A set of query parameters with which the target is scraped.
-	Params url.Values `yaml:"params,omitempty"`
+	Params map[string]string `yaml:"params,omitempty"`
```

Comment:
My expectation is that you'd either want multiple params or the ability to relabel a given single param but rarely both (and if you do want to pass in multiple you always have to option of what's on the other end handling separating out the params).

Number would be `__param_`, `__param2_`, `__param3_` etc. if we ever need it.